frameworks/helloworld/src/main/dist/examples/config_update.yml

```diff
@@ -0,0 +1,90 @@
+name: {{FRAMEWORK_NAME}}
```

Comment:
How do you feel about pulling this into `test_sanity.py` for hello-world? Just need to make one of the pods multistep, pretty easy. Not sure if multistep warrants it's own setup/teardown.